wp-content/themes/core/components/routes/search/css/search.pcss

```diff
@@ -0,0 +1,114 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Route: Search Results
+ *
+ * ----------------------------------------------------------------------------- */
+
+.search-results__container {
+	max-width: var(--grid-width-staggered-double);
+	margin: 0 auto;
+	padding-top: var(--spacer-70);
+	padding-bottom: var(--spacer-70);
+}
+
+.search-results__title {
+	margin-bottom: var(--spacer-20);
+
+	@media (--viewport-medium) {
+		margin-bottom: var(--spacer-30);
+	}
+}
+
+.search-results__form {
+	margin-bottom: var(--spacer-40);
+
+	@media (--viewport-medium) {
+		margin-bottom: var(--spacer-70);
+	}
+}
+
+/* -----------------------------------------------------------------------------
+ * Search Results: Summary
+ * ----------------------------------------------------------------------------- */
+
+.search-results__summary {
+	margin-bottom: var(--spacer-20);
+	line-height: 1.75;
+	font-weight: 700;
+	letter-spacing: 0.1px;
+
+	&:last-child {
+		margin-bottom: 0;
+	}
+}
+
+/* -----------------------------------------------------------------------------
+ * Search Results: Items
+ * ----------------------------------------------------------------------------- */
+
+.search-result {
+	padding-top: var(--spacer-30);
+	padding-bottom: var(--spacer-30);
+	border-top: 1px solid var(--color-neutral-10);
+	cursor: pointer;
+
+	&:last-child {
+		border-bottom: 1px solid var(--color-neutral-10);
+	}
+
+	@media (--viewport-medium) {
+		display: flex;
+		padding-top: var(--spacer-40);
+		padding-bottom: var(--spacer-40);
+	}
+}
+
+.search-result__body {
+	@media (--viewport-medium) {
+		padding-right: var(--spacer-30);
+	}
+
+	@media (--viewport-full) {
+		padding-right: var(--spacer-50);
+	}
+}
+
+.search-result__title {
+	@mixin t-input;
+	margin-bottom: var(--spacer-10);
+	font-weight: 700;
+
+	@media (--viewport-full) {
+		@mixin t-display-x-small;
+		margin-bottom: 0;
+	}
+}
+
+.search-result__excerpt {
+	margin-bottom: var(--spacer-20);
+
+	@media (--viewport-medium) {
+		margin-bottom: var(--spacer-30);
+	}
+}
+
+.search-result__cta > a {
+	@mixin t-body-small;
+	word-break: break-all; /* Break long URLs */
+
+	&:focus,
+	&:hover,
+	.search-result:hover & {
+		color: var(--color-primary);
+	}
+}
+
+.search-result__image {
+	display: none;
+
+	@media (--viewport-medium) {
+		display: block;
+		width: 168px;
+		flex-shrink: 0;
+	}
+}
```

Comment:
These can all be part of the a specific card style/variation once the loop-item template is consolidated into a card style.